src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -2409,7 +2409,22 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::SetChannelMaxTransmitPow
 template <typename InterfaceType, typename ProcessContextType>
 otError RadioSpinel<InterfaceType, ProcessContextType>::SetRadioRegion(uint16_t aRegionCode)
 {
-    return Set(SPINEL_PROP_PHY_REGION_CODE, SPINEL_DATATYPE_UINT16_S, aRegionCode);
+    otError error;
+
+    error = Set(SPINEL_PROP_PHY_REGION_CODE, SPINEL_DATATYPE_UINT16_S, aRegionCode);
+
+    if (error == OT_ERROR_NONE)
+    {
+        otLogNotePlat("Set region code \"%c%c\" successfully", static_cast<char>(aRegionCode >> 8),
```

Comment:
It is documented that `aRegionCode` is always formatted as two ascii chars? 
Should we  log it a hex `0x%04x` value (as well)?

Reply:
This is set from the [url](https://github.com/openthread/openthread/blob/main/src/posix/platform/radio.cpp#L130), we expect it to be the [alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).

I agree we should make it clear of this definition.

Reply:
The function `otPlatRadioSetRegion` will also call this function to set the region code.
The function [otPlatRadioSetRegion](https://github.com/openthread/openthread/blob/main/include/openthread/platform/radio.h#L1091) has declared that the region code is 2-bytes ascii.  

After I update the definition of the region code in the [url](https://github.com/openthread/openthread/blob/main/src/posix/platform/radio_url.cpp#L92), I think we don't need to show the hex value. @abtink Thoughts?

Reply:
SGTM.